torchaudio/functional.py

```diff
@@ -801,3 +801,74 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
     return torch.nn.functional.conv1d(
         specgram, kernel, groups=specgram.shape[1] // specgram.shape[0]
     ) / denom
+
+
+def gain(waveform, gain_db):
+    r"""Apply amplification or attenuation to the audio signal.
+
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time)
+       gain_db(float, default=0.0) Gain adjustment in decibels (dB).
+
+    Returns:
+       torch.Tensor: waveform amplified by gain_db
+    """
+
+    # gain_db = 20log(Tensor_output/tensor_input
+    ratio = 10 ** (gain_db / 20)
+
+    return waveform * ratio
+
+
+def scale_to_interval(waveform, interval=1.0):
+    r"""Scales waveform to an interval.
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time)
+       interval (float): The bounds of the interval, where the float indicates
+           the upper bound and the negative of the float indicates the lower
+           bound (Default: `1.0`)
+           Example: interval=1.0 -> [-1.0, 1.0]
+    Returns:
+       torch.Tensor: waveform scaled to interval
+    """
+    abs_max = torch.max(torch.abs(waveform))
+    ratio = abs_max / interval
+    waveform /= ratio
+
+    assert torch.max(torch.abs(waveform)) == interval
+
+    return waveform
+
+
+def channels(waveform, channel):
```

Comment:
Thanks for looking at this :) As noted in the body of the [PR](https://github.com/pytorch/audio/issues/260#issue-486504615), `channels` doesn't need its own function.
* If it's used to inform `torchaudio.load` of the number of channels in a raw audio file, this would go in the `load` function instead.
* If it's used to down mix, then the user can simply compute a mean, or other tensor operations directly.
* If it's used to increase the number of channels, then the user can just use tensor operations to repeat (or use their own algorithm directly).

Reply:
Just to clarify: from what VCTK use, we need gain and dither, since channels doesn't need its own implementation (see list in #260), "rate" already has "resample".

Reply:
AH! Just saw the note about channels. Will remove!